Userland/Libraries/LibJS/Runtime/AbstractOperations.cpp

```diff
@@ -118,7 +118,7 @@ GlobalObject* get_function_realm(GlobalObject& global_object, FunctionObject con
     // 2. If obj has a [[Realm]] internal slot, then
     if (function.realm()) {
         // a. Return obj.[[Realm]].
-        return function.realm();
+        return &function.global_object();
     }
```

Comment:
Hmm, I feel like the spec and the code don't match each other here..

Reply:
Oh you are fixing it in the next commit. That's what I get for zero lookahead.

Reply:
This is temporary while the function still returns a `GlobalObject`, it's done properly again in the next commit where it's changed to return a `Realm` for the first time. No way around it, unless you want me to squash those two commits :)